db.go

```diff
@@ -68,7 +68,7 @@ func (u AuthUserType) UsernameAsIdentity() bool {
 // It is also responsible for adding new users, changing passwords etc.
 // All operations except for Close must be thread-safe.
 type UserStore interface {
-	Authenticate(identity, password string) error                                                                                  // Return nil error if the username and password are correct, otherwise one of ErrIdentityAuthNotFound or ErrInvalidPassword
+	Authenticate(user string, password string) error                                                                               // Return nil error if the username and password are correct, otherwise one of ErrIdentityAuthNotFound or ErrInvalidPassword
```

Comment:
Also, why are we changing it from identity to user? I think identity is perfectly fine. We also kind of refer to the username/email as identity throughout the code base, so changing the interface would mean it would be inconsistent.